Description: Remove *_LAST_FLAG checks
 The last flag in an enumeration is not really part of the ABI.  Checking it
 here means that changes to parted may unexpectedly break pyparted.
Author: Colin Watson <cjwatson@debian.org>
Bug-Debian: http://bugs.debian.org/697588
Forwarded: no
Last-Update: 2013-01-07

Index: b/src/pydisk.c
===================================================================
--- a/src/pydisk.c
+++ b/src/pydisk.c
@@ -1033,7 +1033,7 @@
         return NULL;
     }
 
-    if ((flag < PED_DISK_FIRST_FLAG) || (flag > PED_DISK_LAST_FLAG)) {
+    if (flag < PED_DISK_FIRST_FLAG) {
         PyErr_SetString(PyExc_ValueError, "Invalid flag provided.");
         return NULL;
     }
@@ -1403,7 +1403,7 @@
         return NULL;
     }
 
-    if ((flag < PED_PARTITION_FIRST_FLAG) || (flag > PED_PARTITION_LAST_FLAG)) {
+    if (flag < PED_PARTITION_FIRST_FLAG) {
         PyErr_SetString(PyExc_ValueError, "Invalid flag provided.");
         return NULL;
     }
